docs/projects/redemptions.md

```diff
@@ -1,7 +1,27 @@
 ---
 id: redemptions
 sidebar_label: Redemptions
-title: Redemptions
+title: 1Hive Redemptions App
 ---
 
-This needs to be written.
+An Aragon app that enables redeeming organizations tokens for organization assets.
+
+<br>
+
+## What is the Redemptions app?
+
+The redemptions app allows your organization to create a vault that exchanges your native token for an underlying asset. For example, the 1Hive native token is HONEY, but often times organization members will want DAI or ETH. The redemptions vault allows BEEs to exchange the HONEY they earn within the organization for the underlying asset of the vault. It also allows anyone to purchase HONEY by depositing the underlying asset (ETH/DAI) in the vault. 
+
+<br>
+
+## What is it good for?
+
+The redpemtions app lets you exchange your organization's native tokens for other tokens such as ETH DAI. This means that anyone can purchase your oganization's token, then use that token to create bounties to open issues that they want to see worked on. For example, 1Hive specializes in resources and mechanisms that support open source commons. Members (BEEs) in the 1Hive organization have skills that are relevant to this. If your project wants to better understand how bonding curves could help you bootstrap a new project, you could purchase HONEY from the redemptions vault, create an open Issue stating your problem and what you think a possible solution might look like, and then BEEs from 1Hive would work on that for you! BEEs who get the HONEY attached to bounties could then exchange it against the redemption vault for ETH or DAI to pay rent or go on vacation. This is great because you get cutting edge research/applications, 1Hive gets to sustainably work on things that further our mission, and the code/results of our work are available for anyone to learn from and utilize. Positive-sum sustainable open source commons :)
```

Comment:
Maybe here as well state that people can't purchase Honey from the `Redemptions` app.

Reply:
Yeah my bad. I was under the impression that the redemptions app was a 2-way vault, but it's actually not. I'll update this section accordingly.